adapted-video-frontend/pom.xml

```diff
@@ -190,7 +197,10 @@
 								</ports>
 
 								<cmd>
-									<shell>java -cp /maven/*:/maven/${project.build.finalName} fr.labri.progress.comet.app.Main --host 0.0.0.0 --port 8080 --rabbit-host $AMQP_PORT_5672_TCP_ADDR --rabbit-port $AMQP_PORT_5672_TCP_PORT --streamer-url $STREAMER_URL </shell>
+									<shell>java -cp /maven/*:/maven/${project.build.finalName}
```

Comment:
docker build shell directive should be on 1 line (otherwhise, docker:build won't work)